src/components/forms/JurisdictionHierachyDownload/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { ErrorMessage, Field, Form, Formik } from 'formik';
+import Papaparse from 'papaparse';
+import React, { useState } from 'react';
+import { Button, FormGroup } from 'reactstrap';
+import JurisdictionSelect from '../../../components/forms/JurisdictionSelect';
+import {
+  DOWNLOAD,
+  DOWNLOAD_FILE,
+  DOWNLOADING,
+  FILE,
+  FILE_DOWNLOADED_SUCCESSFULLY,
+  JURISDICTION_HIERARCHY_TEMPLATE,
+  SELECT_COUNTRY,
+} from '../../../configs/lang';
+import { OPENSRP_V2_SETTINGS, TEXT_CSV } from '../../../constants';
+import { downloadFile, successGrowl } from '../../../helpers/utils';
+import { OpenSRPService } from '../../../services/opensrp';
+import { RawOpenSRPHierarchy, TreeNode } from '../../../store/ducks/opensrp/hierarchies/types';
+import { generateJurisdictionTree } from '../../../store/ducks/opensrp/hierarchies/utils';
+import { LOCATION } from '../../TreeWalker/constants';
+
+export interface JurisdictionHierachyFile {
+  jurisdiction_id: string;
+  jurisdiction_label: string;
```

Comment:
I think that this needs to change to 'jurisdiction_name' which is what is used when [uploading the metadata](https://github.com/onaio/reveal-frontend/blob/fab106acd078fe4f991006d7006154b69647bf55/src/helpers/utils.tsx#L953).